pkg/sink/initialization.go

```diff
@@ -133,3 +135,28 @@ func ConfigureClients(clusterConfig *rest.Config) (Clients, error) {
 		TriggersClient:  triggersClient,
 	}, nil
 }
+
+var elBackoff = wait.Backoff{
+	Duration: 50 * time.Millisecond,
+	Factor:   2.0,
+	Jitter:   0.3,
+	Steps:    10,
```

Comment:
If I'm understanding this correctly, it will try a max of 10 times right? (Not try for a max of 5seconds?)

Reply:
so, its a little of both. the steps is the max number of times, if it ever hits the cap it will set the wait time to that amount and then bail out.

in this case, the first wait time is 50 milliseconds with a scale factor of 2 and jitter of 0.3. The jitter means the scale factor will be somewhere between 1.7 and 2.3. This means that after 10 steps, it waits somewhere around 10 seconds max before it fails

Reply:
makes sense...do you think 10 seconds is quick enough? I was thinking 30s might be a better default?

Reply:
Let me increase the factor and the num steps. Again, this is basically a question of:

`how long will it take for the k8s API server to commit the eventlistener`

I think closer to the 15-20 seconds is appropriate, especially considering that this is a function that blocks the startup of the HTTP endpoint. Let me modify the factor so it ends up around this value